### ISSUE TITLE
[1.10-beta.7] meteor shell .reload no longer works

### ISSUE THREAD

**Issue by hexsprite** [meteor/meteor#10934]
Expected behavior:

That `.reload` in the Meteor shell will restart the Meteor server.

Actual behavior:

```
=> Exited with code: 0
=> Your application is crashing. Waiting for file change
```

Notes:

I wonder if getting rid of the auto-restart feature of Meteor is the cause.

**Comment by hexsprite** [comment]
https://github.com/meteor/meteor/blob/24865b28a0689de8b4949fb69ea1f95da647cd7a/packages/shell-server/shell-server.js#L308

**Comment by benjamn** [comment]
Ok, this should be fixed in `1.10-beta.11` (just released), thanks to a467255d9c5b4bf46da54980aeecd03dbadef351. Please give it a try!

**Comment by hexsprite** [comment]
Yes it works now. Thanks!
